lume_model/models/torch_model.py

```diff
@@ -108,10 +112,18 @@ def validate_output_format(cls, v):
             raise ValueError(f"Unknown output format {v}, expected one of {supported_formats}.")
         return v
 
-    def evaluate(
+    def _set_precision(self, value: torch.dtype):
+        """Sets the precision of the model."""
+        torch.set_default_dtype(value)
```

Comment:
this is probably an overreach, ie. it extends beyond this class/object. Do we need to set the default type for torch here?

Reply:
You're right, that is not needed since we set the model and transformers. I removed it.